engines/freescape/shaders/freescape_cubemap.fragment

```diff
@@ -0,0 +1,9 @@
+OUTPUT
+
+uniform sampler2D skyTexture;
+in vec2 TexCoord;
```

Comment:
This should go before the uniform to match the other shader and what is usually done.
OUTPUT should come after too.

What you have (and is logical) is in, out (so OUTPUT) then uniforms.